tools/crd-ref-docs/config.yaml

```diff
@@ -20,3 +19,10 @@ render:
     - name: LocalPolicyTargetReferenceWithSectionName
       package: sigs.k8s.io/gateway-api/apis/v1alpha2
       link: https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1alpha2.LocalPolicyTargetReferenceWithSectionName
+    - name: Duration
```

Comment:
great! can we do the same things for all the rest types which are ref to gateway-api pkg?

Reply:
- https://github.com/envoyproxy/gateway/blob/02ff9eb749ba7eda1a53dfcbd80fb2b6f46b23fe/api/v1alpha1/backendtrafficpolicy_types.go#L46
- https://github.com/envoyproxy/gateway/blob/02ff9eb749ba7eda1a53dfcbd80fb2b6f46b23fe/api/v1alpha1/basic_auth_types.go#L26
- https://github.com/envoyproxy/gateway/blob/02ff9eb749ba7eda1a53dfcbd80fb2b6f46b23fe/api/v1alpha1/ext_auth_types.go#L70
-